### ISSUE TITLE
Display what Paypal account the store is connected to

### ISSUE THREAD

**Issue by dgoldak** [easydigitaldownloads/easy-digital-downloads#8893]
### Explain your enhancement 
Once you connect to a Paypal Account via PayPal Commerce, there is no information displayed to confirm what Paypal Account you are connected to.

It would be suggested we display the PayPal email address that you are connected too but anything that would be unique to that account would be helpful. For example:
![Markup on 2021-09-29 at 07:38:53](https://user-images.githubusercontent.com/30910807/135269892-c928b2ff-8035-4c3f-9ceb-f615aa11fefb.png)

### Justification or use case
Customer reported that they disconnect from a Paypal Account a connected to a new one. But an order still when though with the old account. In this case there was no way to verify they are connected to the account they are expecting.

**Comment by ashleyfae** [comment]
I really like this idea, but just noting that when I built this, I wanted to display this information by getting it from an API request (hey, here are my API keys, show me which account this is for) but I couldn't find any endpoints that would give me that information.

We might be able to find some other way to get it, but it wasn't as simple as I wanted it to be.

**Comment by mihaijoldis** [comment]
We could display a message like:
Your PayPal Merchant ID is: XXXXXX and then perhaps link the ID to the location in PayPal where you can see it.
https://a.supportally.com/Daztc5+.jpg

**Comment by ashleyfae** [comment]
Yes the merchant ID is something we can expose. It's just frustratingly not all that useful as you then have to look it up in PayPal to compare. I think it's worth double checking (or actually would probably be triple checking lol) we definitely can't get the email anywhere before moving on to other identifiers.
